turbonfs/sample-turbo-config.yaml

```diff
@@ -12,42 +16,75 @@ container: nfsv3test
 #port: 2048
 
 #
-# Auth Config
+# Misc options
+# fuse_* config variables control some fuse configurable value.
+# fuse_max_threads has the same effect as fuse cmdline option "-o max_threads=".
+# fuse_max_idle_threads has the same effect as fuse cmdline option "-o max_idle_threads=".
+# Value of -1 for these imply "use fuse defaults". Fuse defaults must be fine for most cases.
 #
-auth: false
+fuse_max_threads: -1
+fuse_max_idle_threads: -1
+fuse_max_background: 4096
+
+#
+# These are currently not supported.
+#
+#filecache.enable: false
+#filecache.cachedir: /mnt
+#filecache.max_size_gb: 1000
+#cache_max_mb: 4096
+
+#
+# Config controlling misc system behaviour.
+#
+sys.force_stable_writes: true
```

Comment:
On first thought shall we move another section Phase 1.5 
But on second thought for one config, it doesn't worth to create new section.